sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_general_documents_async.py

```diff
@@ -101,20 +106,22 @@ async def analyze_general_documents():
         )
 
         for line_idx, line in enumerate(page.lines):
+            words = line.get_words()
             print(
-                "Line # {} has text content '{}' within bounding box '{}'".format(
+                "...Line # {} has word count {} and text '{}' within bounding box '{}'".format(
```

Comment:
nit: this reads funny to me. maybe "Line # {} has {} words and text..."

Reply:
Ok updated it. It sounded funny to me too, but had decided to just keep the same working as the v2.1 samples. Done.